src/com/rcv/Tabulator.java

```diff
@@ -112,10 +112,10 @@
   }
 
   // map of round to TieBreak objects to record how tiebreaks were decided
-  private Map<Integer, TieBreak> roundToTieBreak = new HashMap<>();
+  private final Map<Integer, TieBreak> roundToTieBreak = new HashMap<>();
```

Comment:
roundToTieBreak is updated here but never queried... should we do anything about this?

Reply:
Yeah, I would just delete this map.